### ISSUE TITLE
TypeError in application menus

### ISSUE THREAD

**Issue by Igazine** [Moonshine-IDE/Moonshine-IDE#1109]
```
: Error #1009
: TypeError: Error #1009
:     at actionScripts.plugin.recentlyOpened::RecentlyOpenedPlugin/openRecentItem()
:     at actionScripts.plugin.recentlyOpened::RecentlyOpenedPlugin/onOpenRecentProject()
:     at flash.events::EventDispatcher/dispatchEvent()
:     at components.views.splashscreen::SplashScreen/handleRecentClick()
:     at flash.events::EventDispatcher/dispatchEvent()
:     at mx.core::UIComponent/dispatchEvent()
:     at components.renderers::RecentProjectRenderer/handleClick()
```

This is related to Haxe conversion. As opposed to ActionScript, Haxe is strictly typed, so if an object that a specific menuitem is based on is not of the required/expected type, these errors occur. I have to recheck/redo relevant parts in the code.

**Comment by JoelProminic** [comment]
I saw this error today when trying to open a Recent Project from the Home tab.   If I test it from `File > Recent Project > %project%` instead, nothing happens and I don't see an error on the console.

**Comment by Igazine** [comment]
This has been fixed. Please wait for the master merge before testing, I'll mark this issue appropriately with the required label

**Comment by Igazine** [comment]
Changes have been merged into master

**Comment by JoelProminic** [comment]
I confirmed this is fixed when opening a recent project or file from the Home tab.

I'm still seeing that the File > Recent Projects and File > Recent Files actions do not do anything, but I think this is a separate issue.

(FYI, if you are wondering why the issue closed earlier, putting "Fixes #1109" in a commit now closes the indicated issue - we saw this behavior in another issue)
